test/test.js

```diff
@@ -808,6 +808,28 @@ describe('node-fetch', function() {
 		});
 	});
 
+	it('should not support bad https request', function() {
+		this.timeout(5000);
+		url = 'https://192.30.252.128/';
```

Comment:
This is the github IP. Couldn't think of any other way to test a bad/missing cert.